db/migrations/20190606092441-create-user.js

```diff
@@ -21,7 +25,7 @@ module.exports = {
       },
       role: {
         type: Sequelize.ENUM,
-        values: ['None', 'Admin', 'SuperAdmin', 'Student', 'Teacher'],
+        values: _.valuesIn(konst.role),
```

Comment:
does this do the same thing?

Reply:
yea, the only difference is that constant strings are now stored in separated file - konst.toml